apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -48,19 +49,59 @@ ${context}
  *
  * @param client The OpenAI client to use.
  * @param prompt The user's question generation prompt.
+ * @param promptUserInput The user's indication of how to create student input boxes.
+ * @param mandatoryElementIds Elements that we must pull documentation for.
  * @param authnUserId The user's authenticated user ID.
  * @returns A string of all relevant context documents.
  */
-async function makeContext(client: OpenAI, prompt: string, authnUserId: string): Promise<string> {
+async function makeContext(
```

Comment:
Five arguments is the point at which you might consider using an options object with named properties instead of using positional arguments.

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483